app/views/shared/_errors.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if obj.errors.any? %>
+  <div role="alert">
+    <h4 class="alert-heading">The following errors prevented the <%= obj.class.name.downcase %> from being saved</h4>
+    <ul>
+      <% obj.errors.full_messages.each do |msg| %>
+        <li><%= msg %></li>
+      <% end %>
+    </ul>
+    <button >
+      <span >&times;</span>
+    </button>
+  </div>
+<% end %>
```

Comment:
eto ung .erb ni Errors.
Si "obj" is a variable, i-indicate siya sa ibang pages (log-in or signup) para gumana.
![image](https://user-images.githubusercontent.com/86232999/145023305-60fe5c68-368c-4f69-b85f-ca8d6d9499f2.png)